scripts/tests/test_safe_template_linter.py

```diff
@@ -36,9 +36,6 @@ def test_process_os_walk_with_includes(self):
 
         output = out.getvalue()
         self.assertIsNotNone(re.search('test\.html.*mako-missing-default', out.getvalue()))
```

Comment:
@efischer19 I brought back this test and just removed the include violations below.